homeassistant/components/ddns_ovh/__init__.py

```diff
@@ -0,0 +1,135 @@
+"""The Dynamic DNS for OVH integration."""
+import asyncio
+import base64
+from datetime import datetime, timedelta
+import logging
+
+import aiohttp
+from aiohttp.hdrs import AUTHORIZATION
+import async_timeout
+import voluptuous as vol
+
+from homeassistant.const import CONF_DOMAIN, CONF_PASSWORD, CONF_TIMEOUT, CONF_USERNAME
+from homeassistant.core import Event, HomeAssistant
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.event import async_track_time_interval
+from homeassistant.helpers.typing import ConfigType
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = "ddns_ovh"
+
+INTERVAL = timedelta(minutes=1)
+
+DEFAULT_TIMEOUT = 10
+
+UPDATE_URL = "https://www.ovh.com/nic/update"
+
+CONFIG_SCHEMA = vol.Schema(
```

Comment:
For integrations that connect to devices or services, we no longer accept new YAML configuration or changes.

This integration needs to be refactored first to use a configuration flow and config entries first.

More information about this can be found in Architecture Decision Record:

- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

See our developer documentation on how to get started creating a configuration flow for this integration:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

As these changes often involve a bit of work and some significant shift in the current code, we will close this PR for now.

Thanks already! :+1: